### DISCUSSION TITLE
How can I create a configuration that can be shared by all users on the same machine?

### DISCUSSION THREAD

**Discussion by SadSock** [neovim/neovim#30809] [Q&A]
As our company's development machine is offline and installing third-party plugins is cumbersome, can we establish a universal configuration that all users can access, streamlining the process and avoiding individual setups?

**Comment by zeertzjq** [comment]
This depends on how you install Nvim. On some Linux distros this can be achieved by adding files under `/usr/share/nvim/` or `/usr/share/nvim/site/`.

**Reply by SadSock** [reply]
i install neovim using  “apt install neovim”

**Comment by dundargoc** [comment]
`:h sysinit.vim`
